CHANGELOG.md

```diff
@@ -9,6 +9,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 -   Added Filtering for site_no, state, county, dislplay_flag, and national aquifer 
 ### Fixed
 -   Fixed the Delete button styling so button is full height.
+### Changed
+   Changed headers in Well Registry SiteID, Agency, SiteNo, Display Site, In WL Subnetwork?, In QW Subnetwork?, State, Principal Aquifer, Site Name
```

Comment:
These should be complete sentences. I think something like "Updated information shown on the monitoring locations list" would be sufficient.